### PR TITLE
FEAT: Added Video Listing page 

### PR DESCRIPTION
**Pull request by fahd23**
- Added Video Listing page
- Videos On the Home page
- Responsiveness

### CODE REVIEW

File: src/components/singlevideo/SingleVideo.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import ReactPlayer from "react-player";
+import "./singlevideo.css";
+import { videos } from "backend/db/videos";
+import { BiLike } from "react-icons/bi";
+import { FaClock } from "react-icons/fa";
+
+const SingleVideo = ({ id }) => {
+  return videos.map((video) =>
```
Comment (JantuDeb) [inline src/components/singlevideo/SingleVideo.js]:
```
const video = videos.find(video=>video._id === id )

```
then show this in ui, no need to map over video

**Review by JantuDeb — APPROVED** [review]
Overall this is fine, added few comments you can look into

File: src/components/singlevideo/SingleVideo.js (hunk 1 above)
Comment (fahd23) [reply src/components/singlevideo/SingleVideo.js]: Thanks for the suggestion jantu , will do that

**Review by harshalini — APPROVED** [review]
Good to merge

**Pull request merged**